lib/dspace-item.test.ts

```diff
@@ -0,0 +1,469 @@
+/* eslint-disable unicorn/no-null */
+import * as dspaceItem from './dspace-item';
+
+const dspaceResponseObject = {
+  bitstreams: [
```

Comment:
Let's cut this down to only the data needed for the test.

Reply:
Fixed. I didn't do a factory method because this was a small test and we really do want the data structured the way it is. I'll make sure to keep the factory pattern in mind though for other tests.